sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -467,10 +487,24 @@ def fetch_20newsgroups_vectorized(*, subset="train", remove=(), data_home=None,
     with open(join(module_path, 'descr', 'twenty_newsgroups.rst')) as rst_file:
         fdescr = rst_file.read()
 
+    frame = None
+    target_name = ['Category_class', ]
```

Comment:
I would not put a capital letter here if the `feature_names` do not contain capital letter as well.